2022q4/gcc.adoc

```diff
@@ -0,0 +1,26 @@
+=== GCC: Update GCC default version to version 12
+
+Links: +
+link:https://gcc.gnu.org[GCC Project] URL: link:https://gcc.gnu.org[https://gcc.gnu.org] +
+link:https://gcc.gnu.org/gcc-11/[GCC 11 release series] URL: link:https://gcc.gnu.org/gcc-11/[https://gcc.gnu.org/gcc-11/] +
+link:https://gcc.gnu.org/gcc-12/[GCC 12 release series] URL: link:https://gcc.gnu.org/gcc-12/[https://gcc.gnu.org/gcc-12/]
+
+Contact: Lorenzo Salvadore <salvadore@FreeBSD.org> +
+
+ * GCC default version has now been updated to version 12.
+   Thank you very much to antoine@ for running the necessary exp-runs and to all the contributors, maintainers and committers involved in the process.
+   As it has been noted last quarter, for every major version of GCC FreeBSD usually waits for the release of the second minor version to update GCC default version. However next time I would like to attempt to update the default version as soon as the first major version of GCC 13 is out. The rationale for waiting the second minor release is to wait for other operating systems (in particular Linux distributions) to find the bugs, report them and fix them, so that FreeBSD could focus mainly on FreeBSD specific cases. But this also means that upstream softwares only hear from FreeBSD rarely and mostly when it concerns FreeBSD only, thus our operating system risks to be considered as minor and unimportant for them.
```

Comment:
In "as soon as the first major version of GCC 13 is out", do you mean 13.0? If you do, you probably need to s/major version/minor release/ (or just simplify to "as soon as GCC 13.0 is out").

In "find the bugs, report them and fix them" you need another comma before "and".

In "upstream softwares only hear from FreeBSD", either s/softwares/software/ or s/softwares/software developers/.

Reply:
Yes, I meant from the first minor version (which is 13.1: GCC starts counting minor versions from .1 and .0 is used for development versions only).